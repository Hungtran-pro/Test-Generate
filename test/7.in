22
109
61
68
93
60
111
112
116
119
99
88
101
98
103
100
114
87
103
90
77
114
112
