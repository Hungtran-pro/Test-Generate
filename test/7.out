612955947
312450027
599807473
482361088
266177107
466486836
597825754
670858904
942906793
623131616
142382092
351513474
892187321
981043477
742820472
998432931
241802082
981043477
2991045
216191719
998432931
597825754
